proto/cosmos/sanction/v1beta1/sanction.proto

```diff
@@ -13,15 +14,23 @@ message Params {
   // If this is zero, immediate sanctioning is not available.
   // Otherwise, if a sanction governance proposal is issued with a deposit at least this large, a temporary sanction
   // will be immediately issued that will expire when voting ends on the governance proposal.
-  repeated cosmos.base.v1beta1.Coin immediate_sanction_min_deposit = 1
-      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+  repeated cosmos.base.v1beta1.Coin immediate_sanction_min_deposit = 1 [
+    (gogoproto.nullable)     = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (amino.dont_omitempty)   = true,
+    (amino.encoding)         = "legacy_coins"
```

Comment:
Why are these needed, and do we even use amino encoding anymore?

Reply:
I'm not entirely sure. I added them here because the SDK added them to all their `Coins` fields, but I couldn't find anything that indicated why they were added or what exactly they do.  The `amino.proto` documentation makes it sound like it's related to JSON encoding, though.

I took these lines out and regenerated stuff, and the only thing that changed was the `fileDescriptor_...` byte slice which is pretty much just the contents of that file. But that's also the case with options like `(cosmos_proto.scalar) = "cosmos.AddressString"` or `(cosmos.msg.v1.signer)`.

I did some testing on two versions of a proto message, one with these options, one without. I generated JSON from each in three different ways:`encoding/json.Marshal`, `"github.com/cosmos/cosmos-sdk/codec".Codec#MarshalJSON` and `"github.com/cosmos/cosmos-sdk/codec".LegacyAmino#MarshalJSON`. I used a fully set-up encoding config for the latter two. All six combos produced the same json.

So either:
1. We don't need these options.
2. Or we're not wiring things up right, causing these to not have meaning.
3. Or they affect something else.